### PR TITLE
feat(logs): [OWL-25156122] Add handler for guildMemberRemove and update logEvent function

### PR DESCRIPTION
**Pull request by devthena**

### CODE REVIEW

File: src/discord/events/guildMemberAdd.ts [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
-    if(!ID || !welcomeRole) return;
+  if (!ID || !welcomeRole) return;
+  if (member.roles.cache.some(roles => roles.id === ID)) return;
 
-    // if user already has the role, do nothing
-    if(message.member?.roles.cache.some(roles => roles.id === ID)) return;
-
-    message.member?.roles.add(welcomeRole?.id || ID)
-}
+  member.roles.add(welcomeRole.id || ID).then(() => {
```
Comment (alvinb) [inline src/discord/events/guildMemberAdd.ts:19]: should we also add a `.catch` if it fails?

Comment (devthena) [reply src/discord/events/guildMemberAdd.ts:19]: yeah that's a good point. i'll make the update!

File: src/constants/config.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ export const CONFIG = {
     LOGS: {
       ACTIVITIES: '1012123780132372570',
       ALERTS: '1012124179371405353',
-      TIMEOUTS: '1012123821580496966',
+      LEAVERS: '1012123821580496966',
       USERS: '1022346939502698566',
```
Comment (alvinb) [inline src/constants/config.ts:12]: are the numbers here IDs?

**Review by alvinb — APPROVED** [review]

File: src/constants/config.ts (hunk 2 above)
Comment (devthena) [reply src/constants/config.ts:12]: yeah they are channel ids

**Pull request merged**